worlds/sc2/item/item_names.py

```diff
@@ -566,6 +566,8 @@
 HUNTER_KILLERS       = "Hunter Killers"
 TORRASQUE_MERC       = "Torrasque"
 HUNTERLING           = "Hunterling"
+YGGDRASIL       = "Yggdrasil"
```

Comment:
Keep the code aligned for easier reading

Reply:
Fixed